packages/dara-components/js/common/table/table.tsx

```diff
@@ -401,23 +401,11 @@ function Table(props: TableProps): JSX.Element {
 
     /**
      * Returns a row by index
-     * - first tries to retrieve it from the loader data
-     * - if not found, tries to retrieve it from the extraDataCache
-     * - if not found, fetches a few rows around the index and stores them in the extraDataCache
+     * Fetches a few rows around the index and stores them in the extraDataCache
      *
      * @param idx index of row to get
      */
     async function getRowByIndex(idx: number): Promise<DataRow> {
```

Comment:
Interesting, we previously discussed a solution which would explicitly fetch rows every time, utilizing the existing cache is definitely more efficient. How does it work with sorting and filtering? Does it always yield the right rows because of the cache being cleared when `getData` changes?

Reply:
That was the idea, ended up not fully working on further testing, so added the index as we had previously discussed